src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/contentgrid/view/ContentGrid.kt

```diff
@@ -0,0 +1,85 @@
+package org.wycliffeassociates.otter.jvm.app.ui.contentgrid.view
+
+import de.jensd.fx.glyphs.materialicons.MaterialIcon
+import de.jensd.fx.glyphs.materialicons.MaterialIconView
+import javafx.beans.property.Property
+import javafx.scene.layout.Priority
+import javafx.stage.Screen
+import org.wycliffeassociates.otter.common.data.model.Collection
+import org.wycliffeassociates.otter.common.data.model.Content
+import org.wycliffeassociates.otter.jvm.app.theme.AppStyles
+import org.wycliffeassociates.otter.jvm.app.theme.AppTheme
+import org.wycliffeassociates.otter.jvm.app.ui.contentgrid.viewmodel.ViewContentViewModel
+import org.wycliffeassociates.otter.jvm.app.widgets.card.DefaultStyles
+import org.wycliffeassociates.otter.jvm.app.widgets.card.card
+import tornadofx.*
+
+
+class ContentGrid : Fragment() {
+    private val viewModel: ViewContentViewModel by inject()
+
+    val activeCollection: Property<Collection> = viewModel.activeCollectionProperty
+    val activeProject: Property<Collection> = viewModel.activeProjectProperty
+    val activeContent: Property<Content> = viewModel.activeContentProperty
+
+    init {
+        importStylesheet<ContentGridStyles>()
+        importStylesheet<DefaultStyles>()
+    }
+
+    override val root = stackpane {
+        hgrow = Priority.ALWAYS
+        vgrow = Priority.ALWAYS
+        style {
+            prefWidth = Screen.getPrimary().visualBounds.width.px - 50.0
+            prefHeight = Screen.getPrimary().visualBounds.height.px - 50.0
+        }
+        addClass(AppStyles.appBackground)
+        hbox {
+
+            vbox {
+                hgrow = Priority.ALWAYS
+                vbox {
```

Comment:
why hbox { vbox { vbox { ?

Reply:
i think that was just something that i missed from refactoring. it's not needed